### DISCUSSION TITLE
How to reduce high cpu utilisation

### DISCUSSION THREAD

**Discussion by dillipkumarpradhan** [reportportal/reportportal#1635] [Q&A]
> I have deployed my RP in a ec2 Instance of Type T3Large

<img width="568" alt="Screenshot 2022-01-11 at 5 25 15 PM" src="https://user-images.githubusercontent.com/32538972/148938274-04f97634-b45d-48ec-a47b-4da9c02147ef.png">

There is enough space on my machine But cpu utilisation is very high =~90% because of java process

**12676 ec2-user  20   0 3659888   1.3g 174416 S 87.4 17.5   0:10.91 java**
87.4  is the % of CPU consumed by your java process

Any way that that we can reduce the cpu utilization

**Comment by HardNorth** [comment]
Can you provide mode details? Which exact command launched the process?

**Reply by dillipkumarpradhan** [reply]
Have Downloaded the docker compose file  and ran docker-compose -p reportportal up -d --force-recreate

**Comment by evjlobanova** [comment]
@irynakozak2  Iryna, could you please check the thread.

@dillipkumarpradhan a few links to configure RP performance
https://reportportal.io/docs/Optimal-Performance-Hardware
https://reportportal.io/docs/Basic-monitoring-configuration
https://reportportal.io/docs/How-to-clean
